BUILD

```diff
@@ -2743,9 +2758,11 @@ grpc_cc_library(
     ],
     tags = ["nofixdeps"],
     visibility = ["@grpc:grpcpp_gcp_observability"],
```

Comment:
Why do you need this?

Reply:
We need to define this macro to use openssl instead of boringssl. When USE_OPENSSL is set to 1, openssl will be used else boringssl will be used.